src/ptxview.cpp

```diff
@@ -1379,7 +1382,11 @@ int main(int argc, char **argv)
     glutPassiveMotionFunc(mousePassiveEvent);
     glutKeyboardFunc(keyboardEvent);
     glutSpecialFunc(keySpecEvent);
-    
+
+#ifdef WIN32
+    glewInit();
+#endif
+
```

Comment:
Why does it need glewInit(), and only on Windows?

Reply:
On Windows, some API is not declared in the GL/gl.h header. So we need to include these declarations using the GLEW library and get the implementation from the driver using glewInit.

Reply:
If do not include GLEW, the following error occurs on Windows 
![image](https://user-images.githubusercontent.com/33077106/176708817-9d759def-1ae1-40af-a0d6-baca02988860.png)